backend/src/schema/resolvers/notifications.spec.js

```diff
@@ -52,80 +59,220 @@ describe('currentUser { notifications }', () => {
         }
         await Promise.all([
           factory.create('User', neighborParams),
-          factory.create('Notification', { id: 'not-for-you' }),
-          factory.create('Notification', { id: 'already-seen', read: true }),
+          factory.create('Notification', {
+            id: 'post-mention-not-for-you',
+          }),
+          factory.create('Notification', {
+            id: 'post-mention-already-seen',
+            read: true,
+          }),
+          factory.create('Notification', {
+            id: 'post-mention-unseen',
+          }),
+          factory.create('Notification', {
+            id: 'comment-mention-not-for-you',
+          }),
+          factory.create('Notification', {
+            id: 'comment-mention-already-seen',
+            read: true,
+          }),
+          factory.create('Notification', {
+            id: 'comment-mention-unseen',
+          }),
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-08-19T14:16:15Z" title="Monday, August 19th 2019, 4:16:15 pm +02:00">Aug 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -52,80 +59,220 @@ describe('currentUser { notifications }', () => {
         }
         await Promise.all([
           factory.create('User', neighborParams),
-          factory.create('Notification', { id: 'not-for-you' }),
-          factory.create('Notification', { id: 'already-seen', read: true }),
+          factory.create('Notification', {
+            id: 'post-mention-not-for-you',
+          }),
+          factory.create('Notification', {
+            id: 'post-mention-already-seen',
+            read: true,
+          }),
+          factory.create('Notification', {
+            id: 'post-mention-unseen',
+          }),
+          factory.create('Notification', {
+            id: 'comment-mention-not-for-you',
+          }),
+          factory.create('Notification', {
+            id: 'comment-mention-already-seen',
+            read: true,
+          }),
+          factory.create('Notification', {
+            id: 'comment-mention-unseen',
+          }),
```


I love these IDs! :smile:

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-20T05:27:43Z" title="Tuesday, August 20th 2019, 7:27:43 am +02:00">Aug 20, 2019</time>_

---


Inspired by the earlier ones by @roschaefer … @alina-beck 🤗